lesson13/src/homework.js

```diff
@@ -0,0 +1,119 @@
+// task 1
+const nestedArray = [[1, 2], [3, [4, 5]], 6]
+function flattenArray(array) {
+  return array.flat(Infinity)
+}
+console.log(flattenArray(nestedArray))
+
+// task 2
+const products = [
+  { id: 1, name: 'Телефон', inStock: true },
+  { id: 2, name: 'Ноутбук', inStock: false },
+  { id: 3, name: 'Часы', inStock: false }
+]
+function isAnyProductInStock(product) {
```

Comment:
```suggestion
function isAnyProductInStock() {
```

параметр не используется